runbotics/runbotics-orchestrator-ui/src/src-app/views/credentials/Credential/EditCredential/CredentialAttribute/Attribute.styles.ts

```diff
@@ -9,14 +9,14 @@ export const StyledGridContainer = styled(Grid)(
     `
 );
 
-export const StyledAttributeCard = styled(Card)<{ isEditMode: boolean }>(
-    ({ theme, isEditMode }) => `
+export const StyledAttributeCard = styled(Card)<{ iseditmode: string }>(
```

Comment:
ok, I can change it. changing it was because there was an error in the console that actual DOM element can have only lover cased properties. stupid, but still.